fairmq/zeromq/FairMQMessageZMQ.cxx

```diff
@@ -12,60 +12,71 @@
  * @author D. Klein, A. Rybalchenko, N. Winckler
  */
 
-#include <cstring>
-#include <cstdlib>
 
 #include "FairMQMessageZMQ.h"
```

Comment:
Although this is correct usage of `""`, I would use `#include <zeromq/FairMQMessageZMQ.h>`, or later, once we refactored all classes to namespaces `#include <fairmq/zeromq/Message.h>`

Reply:
I would rather do it as a separate PR, together with the namespace changes. For now I apply it only to new includes.